driver_cpl/cime_config/config_component.xml

```diff
@@ -519,6 +519,15 @@
      <desc>The environment variables that will be loaded for this machine</desc>
   </entry>
 
+  <entry id="BATCH_SYSTEM">
```

Comment:
We now only define one batch related variable here, all of the others are in config_batch.xml